Address some reviewer comments

jouvin opened the pull request:
Comments attributed to MJ in https://docs.google.com/document/d/1y1xrJxPkJcGxpuF_7NrCfCqLiy4b4Ahg2zZoQ9SWNCk/edit

eduardo-rodrigues commented on CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex:
```diff
@@ -352,7 +352,7 @@ \section{Software and Computing
 specialised processing units dedicated to particular tasks, but with
 possibly large parts of the device switched off most of the time,
-so-called dark silicon.
+so-called dark silicon~\cite{IntelCSA}.
```
@graeme-a-stewart also added a citation at this place, albeit a different one. Would it make sense to have both? I confess I did not compare the content both links point to.

jouvin replied:
I think so... There will probably be a conflict when of the PR is merged but it should be easy to fix.

eduardo-rodrigues replied:
OK, great. Thanks.

graeme-a-stewart replied:
Hmmm, I am not sure we should cite something that doesn't exist yet, except in Intel's head. In my PR I added a much more robust citation to dark silicon in general.

jouvin replied:
I saw your citation and I think it is great. I have no problem to remove mine, I added it because @davidlange6 commented that it was an interesting reference. I would not think it is necessarily a bad reference to illustrate potentially more disruptive architectures to use something about the ideas being investigated by big players...

jouvin commented:
I addressed the remarks concerning SDN, including adding an item in the roadmap. Check by others is welcome!

I'm still waiting the answer from Romain and Hannah about references to "next-generation identity federation", reason why this isssue is marked `WIP`. But no problem to merge it if you prefer and we can address the last point in another PR.

davidlange6 commented:
hi all - since we have 3 prs that will all conflict, I propose to merge this one. It looks to be ready to go (and is simple)

eduardo-rodrigues commented:
Agreed, ready to be merged.
